### PR TITLE
Update README.md on SlotFills to add info + example about the prop `fillProps`

### PR DESCRIPTION
**Pull request by rmorse**
Add info and example about `fillProps` prop on `Slot`

### CODE REVIEW

File: packages/components/src/slot-fill/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Both `Slot` and `Fill` accept a `name` string prop, where a `Slot` with a given
 
 `Slot` with `bubblesVirtually` set to true also accept an optional `className` to add to the slot container.
 
-`Slot` also accepts optional `children` function prop, which takes `fills` as a param. It allows to perform additional processing and wrap `fills` conditionally.
+`Slot` accepts optional `children` function prop, which takes `fills` as a param. It allows to perform additional processing and wrap `fills` conditionally.
```
Comment (ndiego) [inline packages/components/src/slot-fill/README.md]:
```suggestion
`Slot` accepts an optional `children` function prop, which takes `fills` as a param. It allows you to perform additional processing and wrap `fills` conditionally.
```

File: packages/components/src/slot-fill/README.md [hunk 2]
```diff
@@ -89,3 +89,27 @@ const Toolbar = ( { isMobile } ) => (
 	</div>
 );
 ```
+
+Props can also be passed from a `Slot` to a `Fill` by using the prop `fillProps` on the `Slot`:
+```jsx
```
Comment (ndiego) [inline packages/components/src/slot-fill/README.md]:
```suggestion
Props can also be passed from a `Slot` to a `Fill` by using the prop `fillProps` on the `Slot`:

```jsx
```

**Review by ndiego — APPROVED** [review]

**Comment by ndiego** [comment]
Thanks @rmorse, this looks great. Just a few minor tweaks.

**Comment by rmorse** [comment]
Good catch @ndiego - commited the changes, thanks!

**Review by mburridge — APPROVED** [review]
LGTM

**Pull request merged**